packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/sku_details_wrapper.dart

```diff
@@ -68,8 +72,17 @@ class SkuDetailsWrapper {
   final String introductoryPrice;
 
   /// [introductoryPrice] in micro-units 990000
-  @JsonKey(name: 'introductoryPriceAmountMicros', defaultValue: '')
-  final String introductoryPriceMicros;
+  @JsonKey(name: 'introductoryPriceAmountMicros', defaultValue: 0)
+  final int introductoryPriceAmountMicros;
+
+  /// String representation of [introductoryPrice] in micro-units 990000
+  @Deprecated('Use `introductoryPriceAmountMicros` instead.')
+  @JsonKey(ignore: true)
+  String get introductoryPriceMicros => _introductoryPriceMicros.isEmpty
+      ? introductoryPriceAmountMicros != 0
```

Comment:
I didn't get why we needed the nested part. If introductoryPriceAmountMicros is 0, can't introductoryPriceMicros just be '0'?
Wouldn't the below work?
```dart
String get introductoryPriceMicros => _introductoryPriceMicros.isEmpty ? 
                                                                  introductoryPriceAmountMicros.toString() :
                                                                  ''
```

Reply:
Yes it would work, but the original behavior (although wrong) was to return an empty string as default value. I tried to stick as close as possible to that using the nested condition.

Reply:
If returning empty string is wrong, we don't need to worry about the wrong behavior. This PR suppose to "fix" it :)

Reply:
Yes that makes sense. In that case the inner condition can indeed be removed.

Reply:
Done.